metadata-foundationdb/metadata-foundationdb.c

```diff
@@ -3,10 +3,15 @@
 #define FDB_API_VERSION 620
 #include <foundationdb/fdb_c.h>
 
+#include <pthread.h>
+#include <metadata.h>
+
 #include "metadata-foundationdb.h"
 
 uint64_t metadata_key = 0ULL;
 
+FDBDatabase *foundationdb_database = NULL;
```

Comment:
should we prefix this with `g_`?

Reply:
I originally took that prefix from some of the glib example code.

Looking up visibility rules in C, it seems like the best option is to declare it static - this should restrict visibility to the compilation unit, eg. metadata-foundationdb.c itself.